.github/workflows/linters.yml

```diff
@@ -133,7 +133,7 @@ jobs:
 
       - name: Check exported files
         run: |
-          EXPECTED="LICENSE,README.md,autoload.php,composer.json"
+          EXPECTED="LICENSE,README.md,autoload.php,composer.json,docker/,docker/unstable_cluster/,docker/unstable_cluster/Dockerfile,docker/unstable_cluster/create_cluster.sh,docker/unstable_cluster/docker-compose.yml,docker/unstable_cluster/redis.conf"
```

Comment:
can this be asterisked - ie docker/* or docker/**? Does that work here?

Reply:
I think we can add docker folder as exclusion, would be even better